pkg/deployd/deployd/deployd.go

```diff
@@ -138,7 +140,13 @@ func Run(logger *log.Entry, req *pb.DeploymentRequest, cfg config.Config, kube k
 	errors := make(chan error, len(resources))
 
 	for index, resource := range resources {
-		addCorrelationID(&resource, req.GetDeliveryID())
+		AddAnnotations(
+			&resource,
+			&(map[string]string{
+				kubeclient.CorrelationIDAnnotation: req.GetDeliveryID(),
+				kubeclient.GitRefShaAnnotation:     req.GetGitRefSha(),
```

Comment:
Perhaps also add the `GitRef` annotation can be useful here?

Reply:
Done!